wp-pgp-encrypted-emails.php

```diff
@@ -274,6 +291,81 @@ public static function getAdminKey () {
     }
 
     /**
+     * Check validity of S/MIME public certificate.
+     *
+     * @param string $cert
+     *
+     * @return resource certificate|false
+     */
+    public static function checkCert ($cert = null) {
```

Comment:
A minor thing, but this function seems to be using a two-space indentation, but the other parts of this file use a four-space indention. Let's stick with four spaces as per the WordPress style guide. Thanks! :)